atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/jepsen/events/EventTest.java

```diff
@@ -92,6 +92,44 @@ public void canDeserialiseOkRead() {
     }
 
     @Test
+    public void canDeserialiseFailEventWithStacktrace() {
+        final String exceptionString = new RuntimeException("Error").toString();
```

Comment:
Shouldn't need to add `final` anywhere. Java 8+ automatically adds them where necessary